docs/articles/apex/writing.md

```diff
@@ -2,18 +2,29 @@
 title: Write Apex Code
 ---
 
-View outlines of Apex classes and triggers, see code-completion suggestions, and find syntactic errors in your code. The Apex extension is powered by the [Apex Language Server](language-server).
+Use the Apex extension for VS Code to access code-editing features such as code completion, go to definition, view outlines of Apex classes and triggers, refactoring, and find syntactic errors in your code.
 
-## View Code-Completion Suggestions
+## Code-Completion
 
-To see code-completion suggestions, press Ctrl+space when you’re working in a `.cls` or `.trigger` file. To navigate between the suggestions, use the arrow keys. To auto-complete a suggestion from the list, press Enter. To change how suggestions are pre-selected, see [IntelliSense - Suggestion selection](https://code.visualstudio.com/docs/editor/intellisense).
-![Animation showing code completion of a System.debug() statement](../../images/apex_completion.gif)
+Apex extension provides context-sensitive suggestions when you’re working in an Apex class or trigger. As you type, the auto-completion list of members such as, methods or variables pop up. The auto-completion list also displays the documentation for the suggestions. Use these keys:
```

Comment:
Do we need to start the paragraphs in each section of this doc with "Apex extension"? For example, just say "Context-sensitive suggestions can be provided when working in an Apex class or trigger" or for below, "Code snippets are available for scaffolding class and interface definitions, and a variety of statements such as loops, conditionals, etc."

Reply:
Updated so that all the sections don't repeat 'Apex extension ...'